atd-vzv/src/views/map/InfoBox/MapInfoBox.js

```diff
@@ -11,6 +16,13 @@ const MapInfoBox = ({
   type, // id of feature layer
 }) => {
   const popupInfo = selectedFeature && selectedFeature.properties;
+  const popupX = popupInfo.pixelCoordinates && popupInfo.pixelCoordinates.x;
+
+  const StyledPopup = styled.div`
+    .mapboxgl-popup-content {
+      ${setPopupPosition(popupX)}
+    }
+  `;
```

Comment:
👍 amazing, thanks for showing us how to use styled components via `styled.div`

Reply:
clever 🧠